aptos-move/block-executor/src/view.rs

```diff
@@ -609,15 +633,132 @@ impl<'a, T: Transaction, X: Executable> ParallelState<'a, T, X> {
     }
 }
 
+impl<'a, T: Transaction, X: Executable> ResourceGroupState<T> for ParallelState<'a, T, X> {
+    fn set_raw_group_base_values(&self, group_key: T::Key, base_values: Vec<(T::Tag, T::Value)>) {
+        self.versioned_map
+            .group_data()
+            .set_raw_base_values(group_key.clone(), base_values);
+    }
+
+    fn read_cached_group_tagged_data(
+        &self,
+        txn_idx: TxnIndex,
+        group_key: &T::Key,
+        resource_tag: &T::Tag,
+        maybe_layout: Option<&MoveTypeLayout>,
```

Comment:
ditto in reverse order re:using UnknownOrLayout (but smt we can do later)

Reply:
ditto. - this is correct, and shouldn't be changed